conda_build/build.py

```diff
@@ -582,7 +582,7 @@ def test(m, verbose=True, channel_urls=(), override_channels=False):
         else:
             test_file = join(tmp_dir, 'run_test.sh')
             # TODO: Run the test/commands here instead of in run_test.py
-            cmd = ['/bin/bash', '-x', '-e', test_file]
+            cmd = ['/bin/sh', '-x', '-e', test_file]
```

Comment:
What if there are bashisms in the `build.sh` script of a recipe? We've never made the promise to work with other shells.

I assume BSDs have a package for `bash`, so why not use the path provided to `bash` for each BSD instead of changing the default shell?